src/azure-cli/azure/cli/command_modules/acr/agentpool.py

```diff
@@ -61,11 +61,15 @@ def acr_agentpool_update(cmd,
     _, resource_group_name = validate_managed_registry(
         cmd, registry_name, resource_group_name)
 
+    AgentPoolUpdateParameters = cmd.get_models('AgentPoolUpdateParameters', operation_group='agent_pools')
+
+    update_parameters = AgentPoolUpdateParameters(count=count)
+
     try:
-        return client.update(resource_group_name=resource_group_name,
-                             registry_name=registry_name,
-                             agent_pool_name=agent_pool_name,
-                             count=count)
+        return client.begin_update(resource_group_name=resource_group_name,
```

Comment:
same question above, are we meant to start long running operation without waiting or wait for operation to complete. What does begin_update do? Does it poll for completion?

Reply:
Just name change, no behavior change.